src/app/components/trends/TrendsItem.js

```diff
@@ -40,17 +41,13 @@ const TrendsItem = ({ routeParams, location }) => {
                       team {
                         name
                         dbid
-                        country
```

Comment:
This file was one that I was testing how I might refactor. I ultimately didn't do the refactor here, but I did temporarily move the `TrendsItemComponent` into this file and determine which fields could be safely removed. We still need to move the fragment into `TrendsItemComponent` to really "fix" this but ultimately this at least removes some truly unneeded fields.